portfolio/src/main/webapp/script.js

```diff
@@ -17,7 +17,7 @@
  */
 function addRandomGreeting() {
   const greetings =
-      ['Hello world!', '¡Hola Mundo!', '你好，世界！', 'Bonjour le monde!'];
+      ['“I am Optimus Prime, and I send this message so that our pasts will always be remembered, for in those memories we live on."'];
```

Comment:
If you'd like to, it could be cool to add one or two more quotes. If there's only one quote, then this will always choose that one